doc/setup.md

```diff
@@ -49,13 +49,32 @@ If you are using Windows Subsystem for Linux (WSL) be sure you clone the reposit
 
 These instructions assume that you have [Homebrew](http://brew.sh/) installed.
 
-1. Install dependencies
+1. Install project dependencies
 
-	```
-	brew install supervisor nginx postgresql node \
-	     llvm libomp gsl rust
-	nvm install node
-	```
+Using Homebrew, install core dependencies:
+```
+brew install supervisor
+brew install nginx
+brew install postgresql
+brew install node
+brew install llvm
+brew install libomp
+brew install gsl
+brew install rust
```

Comment:
All of these brew install should be one liners. No need to have a different command for each